### PR TITLE
Support CREATE SEQUENCE

### PR DESCRIPTION
**Pull request by AnEmortalKid**
Supports `CREATE SEQUENCE`

As part of this, I added a `K_GLOBAL` token and noticed that a couple tests started failing. I'm not entirely sure what exactly broke since one of them is related to https://github.com/JSQLParser/JSqlParser/issues/293 , my guess is that the token is now captured and not stored (for the CreateTable case)? 

Partially addresses #112

### CODE REVIEW

**Comment by AnEmortalKid** [comment]
Accidentally formatted most files I touched :( , the diff can be easily viewed with: `?w=1` : https://github.com/JSQLParser/JSqlParser/pull/977/files?w=1

I'll work on reverting the formatting.

File: src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt [hunk 1]
```diff
@@ -532,7 +548,7 @@ Statements Statements() #Statements :
             parseErrors.add(e);
             error_skipto(ST_SEMICOLON);
         }
-        else
```
[the remark this answers is not in the document]
Comment (AnEmortalKid) [reply src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt:535]: I think so, I have a windows computer which is where I did most of this PR. I'll try to `dos2unix` this file and see if a lot changes. I'll dig around and try to get a `.gitattributes` that should prevent this for others.

[the remark this answers is not in the document]
Comment (AnEmortalKid) [reply src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt:535]: Ah, no, that looks like it removed the space after `else `, I opened that up in atom and I think it had the 'remove trailing spaces setting'. I can try to set that back!

File: src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt [hunk 2]
```diff
@@ -4637,4 +4653,112 @@ void readGrantTypes(ArrayList<String> privileges):
...
\ No newline at end of file
+}
+
+Sequence Sequence() #Sequence :
+{
+    String serverName = null, databaseName = null, schemaName = null, sequenceName = null;
+}
+{
+    (
+          LOOKAHEAD(7) serverName=RelObjectName() "." [databaseName=RelObjectName()] "." [sequenceName=RelObjectName()] "." sequenceName=RelObjectName()
```
[the remark this answers is not in the document]
Comment (AnEmortalKid) [reply src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt]: updated, seems much simpler from the parser side!

**Comment by AnEmortalKid** [comment]
@wumpz i think this one might have broken the build :( I don’t see the status check on it! If you wanna hit revert I can check what’s making it fail / try to fix master

**Comment by wumpz** [comment]
@AnEmortalKid I did not yet deploy this version. So if you could provide a correction within today I am fine with this. The problem seems to be that you are introducing multiple new keywords, that are not included in RegObjName parsing. E.g. an Oracle test fails because there is a column name **cycle**.

**Pull request merged**